### PR TITLE
First look at editor's interface iteration

### PR DESCRIPTION
**Pull request by vgezer**
Updated screenshots to 4.3, renamed AssetLib to Asset Library, also changed the structure to ease readability. Removed some detailed stuff and pointed to the other documents since this is only the first look. Detailed project manager screen.

Open for comments.

### CODE REVIEW

**Comment by AThousandShips** [comment]
I'd suggest perhaps splitting this into multiple PRs for each area of the documentation, as it touches on pretty different areas

File: getting_started/introduction/first_look_at_the_editor.rst [hunk 1]
```diff
@@ -6,104 +6,123 @@
...
 .. image:: img/editor_intro_toolbar_2d.webp
 
-Below is the 3D one.
+Below is the 3D one:
 
 .. image:: img/editor_intro_toolbar_3d.webp
 
+.. seealso:: To learn more on workspaces, read :ref:`doc_intro_to_the_editor_interface_four_screens`.
+
+.. seealso:: To learn more on 3D viewport and 3D in general, read :ref:`doc_introduction_to_3d`.
```
Comment (AThousandShips) [inline getting_started/introduction/first_look_at_the_editor.rst:83]:
```suggestion
.. seealso:: To learn more on the 3D viewport and 3D in general, read :ref:`doc_introduction_to_3d`.
```

File: getting_started/introduction/first_look_at_the_editor.rst [hunk 2]
```diff
@@ -6,104 +6,123 @@
...
 .. image:: img/editor_intro_project_templates.webp
 
-You can also change the editor's language by going into the settings menu.
+Project Manager's settings can be opened using the **Settings** menu:
 
 .. image:: img/editor_intro_settings.webp
 
-From here use the language drop down menu to select your language. By default, it is in
-English (EN).
+From here, you can change the editor's language (default is system language), interface theme, display
```
Comment (AThousandShips) [inline getting_started/introduction/first_look_at_the_editor.rst:41]:
```suggestion
From here, you can change the editor's language (default is the system language), interface theme, display 
```

File: getting_started/introduction/first_look_at_the_editor.rst [hunk 3]
```diff
@@ -6,104 +6,123 @@
...
 
-.. seealso:: To learn the Project Manager's ins and outs, read
-             :ref:`doc_project_manager`.
+Once your network mode is set to "online", you can search for demo projects in the open
+source asset library, which includes many projects developed by the community:
 
 .. image:: img/editor_intro_project_templates.webp
 
-You can also change the editor's language by going into the settings menu.
+Project Manager's settings can be opened using the **Settings** menu:
```
Comment (AThousandShips) [inline getting_started/introduction/first_look_at_the_editor.rst:37]:
```suggestion
The Project Manager's settings can be opened using the **Settings** menu:
```

File: tutorials/editor/project_manager.rst [hunk 4]
```diff
@@ -24,24 +32,42 @@ Creating and importing projects
...
+
+Next, the buttons to switch the display type of the folders and files between grid view to list view
+are seen.
 
-When you see the green tick on the right, it means the engine detects an empty
-folder. You can also click the **Create Folder** button to create an empty
-folder based on your project's name.
+Last button on the right will create a new folder.
+
+Favorited folders will be displayed on the left side under **Favorites** section. You can sort the
```
Comment (AThousandShips) [inline tutorials/editor/project_manager.rst:66]:
```suggestion
Favorited folders will be displayed on the left side under the **Favorites** section. You can sort the
```

File: tutorials/editor/project_manager.rst [hunk 5]
```diff
@@ -24,24 +32,42 @@ Creating and importing projects
...
+favorites using the up and down buttons in this section.
+Last chosen folders will be listed under **Recent** list.
```
Comment (AThousandShips) [inline tutorials/editor/project_manager.rst:68]:
```suggestion
Last chosen folders will be listed under the **Recent** list.
```

File: tutorials/editor/project_manager.rst [hunk 6]
```diff
@@ -24,24 +32,42 @@ Creating and importing projects
...
+On the right side of the path field, there are buttons to refresh the current folder's contents,
+favorite/unfavorite the current folder, and show/hide hidden folders.
+
+Next, the buttons to switch the display type of the folders and files between grid view to list view
+are seen.
 
-When you see the green tick on the right, it means the engine detects an empty
-folder. You can also click the **Create Folder** button to create an empty
-folder based on your project's name.
+Last button on the right will create a new folder.
```
Comment (AThousandShips) [inline tutorials/editor/project_manager.rst:64]:
```suggestion
The last button on the right will create a new folder.
```

File: tutorials/editor/project_manager.rst [hunk 7]
```diff
@@ -24,24 +32,42 @@ Creating and importing projects
...
+You can pick a location or type the folder's path in the **Path** field, after choosing a drive.
 
-.. image:: img/editor_ui_intro_project_manager_05.webp
+Left of the path field on the top row contains arrows to navigate back and forward through last 
+visited locations.
+Up arrow navigates to parent folder.
+On the right side of the path field, there are buttons to refresh the current folder's contents,
+favorite/unfavorite the current folder, and show/hide hidden folders.
+
+Next, the buttons to switch the display type of the folders and files between grid view to list view
```
Comment (AThousandShips) [inline tutorials/editor/project_manager.rst:61]:
```suggestion
Next, the buttons to switch the display type of the folders and files between grid view and list view
```
"between" doesn't indicate a one directional relationship, so "and" not "to"

File: tutorials/editor/project_manager.rst [hunk 8]
```diff
@@ -24,24 +32,42 @@ Creating and importing projects
...
 
-Click the **Browse** button to open Godot's file browser and pick a location or type
-the folder's path in the Project Path field.
+Click the **Browse** button to open Godot's file browser.
+You can pick a location or type the folder's path in the **Path** field, after choosing a drive.
 
-.. image:: img/editor_ui_intro_project_manager_05.webp
+Left of the path field on the top row contains arrows to navigate back and forward through last 
+visited locations.
+Up arrow navigates to parent folder.
```
Comment (AThousandShips) [inline tutorials/editor/project_manager.rst:57]:
```suggestion
The up arrow navigates to parent folder.
```

File: tutorials/editor/project_manager.rst [hunk 9]
```diff
@@ -24,24 +32,42 @@ Creating and importing projects
...
 Using the file browser
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Click the **Browse** button to open Godot's file browser and pick a location or type
-the folder's path in the Project Path field.
+Click the **Browse** button to open Godot's file browser.
+You can pick a location or type the folder's path in the **Path** field, after choosing a drive.
 
-.. image:: img/editor_ui_intro_project_manager_05.webp
+Left of the path field on the top row contains arrows to navigate back and forward through last
```
Comment (AThousandShips) [inline tutorials/editor/project_manager.rst:55]:
```suggestion
Left of the path field on the top row contains arrows to navigate backward and forward through the last
```
Or
```suggestion
Left of the path field on the top row contains arrows to navigate backward and forward through recently
```

Either "back and forth" or "backward and forward"

**Comment by AThousandShips** [comment]
Please apply changes in a batch using "add suggestion to batch" to avoid creating a lot of commits

**Comment by vgezer** [comment]
> Please apply changes in a batch using "add suggestion to batch" to avoid creating a lot of commits

Yes, made the same mistake again :).

**Comment by vgezer** [comment]
> I'd suggest perhaps splitting this into multiple PRs for each area of the documentation, as it touches on pretty different areas

I thought about it, but I had to move the text to other relevant documents, not to lose the information. E.g., some text from `first-look-at-interface` was moved to `project-settings`. At the end multiple files were modified.

**Comment by AThousandShips** [comment]
Will give this a second review for content as well soon

File: tutorials/editor/project_manager.rst [hunk 10]
```diff
@@ -89,18 +118,25 @@ Managing projects with tags
...
-Type out the tag name, and click "ok". Your project will now have a tag added to it.
-These tags can be used for any other project in your project manager. In addition tags
-will stay with projects. So if you tag your project, send it to another machine and
-import it into the project manager you will see the tags you created.
+Type out the tag name, and click **OK**. Your project will now have a tag added to it.
+These tags can be used for any other project in your project manager.
+
+To show projects with a specific tag only, you can click on the tags or write ``tag:`` 
+and type the tag you would like to search for in the filter bar. To limit the results 
+using multiple tags, you can click on another tag or add ``tag:``tag: after
```
Comment (AThousandShips) [inline tutorials/editor/project_manager.rst:134]:
```suggestion
using multiple tags, you can click on another tag or add ``tag:`` after 
```

File: tutorials/editor/project_manager.rst [hunk 11]
```diff
@@ -89,18 +118,25 @@ Managing projects with tags
...
+Type out the tag name, and click **OK**. Your project will now have a tag added to it.
+These tags can be used for any other project in your project manager.
+
+To show projects with a specific tag only, you can click on the tags or write ``tag:`` 
+and type the tag you would like to search for in the filter bar. To limit the results 
+using multiple tags, you can click on another tag or add ``tag:``tag: after 
+a space and type another tag in the filter bar.
+
+In addition, tags will stay with projects. So if you tag your project, send it to 
+another machine and import it into the project manager you will see the tags
```
Comment (AThousandShips) [inline tutorials/editor/project_manager.rst:138]:
```suggestion
another machine, and import it into the project manager you will see the tags
```

**Review by AThousandShips — APPROVED** [review]
LGTM

**Review by skyace65 — APPROVED** [review]

**Review by skyace65 — CHANGES_REQUESTED** [review]
This PR needs to be rebased. There's a conflict with proejct_manager.rst

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks!

**Pull request merged**